profiling/src/php_ffi.c

```diff
@@ -105,7 +104,53 @@ void ddog_php_prof_zend_mm_set_custom_handlers(zend_mm_heap *heap,
 #endif
 }
 
-zend_execute_data* ddog_php_prof_get_current_execute_data()
-{
+zend_execute_data* ddog_php_prof_get_current_execute_data() {
     return EG(current_execute_data);
 }
+
+#if PHP_VERSION_ID >= 80000
+static int ddog_php_prof_run_time_cache_handle = -1;
+#endif
+
+void ddog_php_prof_function_run_time_cache_init(const char *module_name) {
+#if PHP_VERSION_ID >= 80000
+    // Grab 2, one for function name and one for filename.
+    ddog_php_prof_run_time_cache_handle =
+        zend_get_op_array_extension_handles(module_name, 2);
```

Comment:
I added this API with 8.0.18 / 8.1.4. If you want to support older versions, i.e. 8.0.0-8.0.17 & 8.1.0-8.1.3, use zend_get_op_array_extension_handle() twice:
```suggestion
#if PHP_VERSION_ID < 80200
        zend_get_op_array_extension_handles(module_name);
    zend_get_op_array_extension_handles(module_name);
#else
        zend_get_op_array_extension_handles(module_name, 2);
#endif
```
which effectively has the same effect.

Reply:
Thanks for the heads ups!